src/index.ts

```diff
@@ -893,9 +906,19 @@ class OpenSIPSJS extends UA {
         call.roomId = roomId
         call.localMuted = false
 
-        this._addCall(call)
+        if (call.direction === 'incoming' && this.autoAnswer) {
```

Comment:
First of all - don't we have enum for call directions?
Secondly,  you can do that simpler:
```js
const doAutoAnswer = call.direction === 'incoming' && this.autoAnswer

this._addCall(call, triggerEvent)
```